native_client/java/standalone.md

```diff
@@ -0,0 +1,193 @@
+# DeepSpeech for standalone Java
+
+### Adding DeepSpeech to your Java project
+>Note: You can do this on your own way too, but this works with the example usage code provided below.
+1. Copy `libdeepspeech.jar` from `./build/` into `{YOUR PROJECT ROOT}/libs/`
+2. Copy '*.so' from `./build/` into `{YOUR PROJECT ROOT}/src/main/resources/jni/x86_64/`
+3. Modify your `build.gradle` file to include:
+```groovy
+plugins {
+   	id 'com.github.johnrengelman.shadow' version '5.0.0' 
+}
+
+repositories {
+    flatDir {
+			dirs 'libs'
+    }
+}
+
+dependencies {
+    compile ':libdeepspeech'
+}
+```
+4. You can now compile your project with `./gradlew shadowJar`.
+
+### Notes
+- At this moment, this will only run on x86-64 Linux.
+
+### Example usage
+`LibDeepSpeech.java` example
+```java
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+
+import org.deepspeech.libdeepspeech.DeepSpeechModel;
+
+//This class facilitates the loading of the native libraries.
+//The implementation in here works with the Gradle guide provided in the README.md document.
+//However you can modify this to whatever you wish.
+public class LibDeepSpeech extends DeepSpeechModel {

```

Comment:
the user should not have to do this, as I explained earlier, it needs to be taken care of in `DeepSpeechModel` class

Reply:
Yep correct. it's no longer up-to-date. Since the DeepSpeechModel class is no longer abstract that can go away. Though the library loading is better off in the hands of the user for non-Android, since it differs per project how it's done. Although if we figure out how to ship the .so files in the libdeepspeech jar, then we can 'standardize' it